tests/ut/ir/operators/test_op_registry.py

```
@@ -323,6 +323,108 @@ def test_matmul_with_transpose_kwargs():
+
+    dim1 = ir.ConstInt(1, DataType.INT32, span)
+    dim16 = ir.ConstInt(16, DataType.INT32, span)
+    dim256 = ir.ConstInt(256, DataType.INT32, span)
+    dim512 = ir.ConstInt(512, DataType.INT32, span)
+
+    lhs = ir.Var("lhs", ir.TensorType([dim16, dim512], DataType.FP32), span)
+    rhs = ir.Var("rhs", ir.TensorType([dim512, dim256], DataType.INT8), span)
+    act_scale = ir.Var("act_scale", ir.TensorType([dim16, dim1], DataType.FP32), span)
+    weight_scale = ir.Var("weight_scale", ir.TensorType([dim1, dim256], DataType.FP32), span)
+
+    with pytest.raises(Exception):
```

Ruff (0.15.20)

[warning] Do not assert blind exception: `Exception`

(B017)